main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        car
+//  action 1:    the car idles
+car.idles();
+//  action 2:    the car accelerates
+car.accelerate();
+//  action 3:    the car turns left
+car.turnLeft(); 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        cat
+//  action 1:    the cat purrs
+cat.purr();
+//  action 2:    the cat meows
+cat.meow();
+//  action 3:    the cat sleeps
+cat.sleep();
 //  3)
+//  Noun:        dog
+//  action 1:    the dog barks
+dog.bark();
+//  action 2:    the dog wags it's tail
+dog.wagTail();
+//  action 3:    the dog runs
+dog.run();
+//  4)           falcon
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  action 1:    the falcon soars
+falcon.soar();
+//  action 2:    the falcon dives
+falcon.dive();
+//  action 3:    the falcon flaps it's wings
+falcon.flapWings();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        lighter    
+//  action 1:    the lighter ignites
+lighter.ignite(); 
+//  action 2:    consumes fuel
+lighter.consumesFuel();
+//  action 3:    creates heat
+lighter.createsHeat();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        controller
+//  action 1:    the controller registers an press
+controller.registerInput();
+//  action 2:    the controller registers a release
+controller.registerRelease();
+//  action 3:    the controller registers a pressure level
+controller.registerPressureLevel();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        amplifier 
+//  action 1:    sends a signal to speaker
+amplifier.sendsSignal();
+//  action 2:    lowers volume
+amplifier.decreaseAmplitude();
+//  action 3:    increase volume
+amplifier.increaseAmplitude();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.